RadixWallet/Features/SettingsFeature/DisplayMnemonics/Coordinator/DisplayMnemonics.swift

```diff
@@ -28,10 +28,12 @@ public struct DisplayMnemonics: Sendable, FeatureReducer {
 	public struct Destinations: Sendable, Equatable, Reducer {
 		public enum State: Sendable, Hashable {
 			case displayMnemonic(DisplayMnemonic.State)
+			case importMnemonicControllingAccounts(ImportMnemonicControllingAccounts.State)
```

Comment:
can't this simply be called `importMnemonic`?

Reply:
we need a complete overhaul of mnemonics anyway, but no, I want the case to reflect which feature is used, `importMnemonic` is another more low level reducer used by that feature